ports/esp32/boards/ESP32_GENERIC_S3/board.json

```diff
@@ -18,6 +18,7 @@
     "url": "https://www.espressif.com/en/products/modules",
     "vendor": "Espressif",
     "variants": {
-        "SPIRAM_OCT": "Support for Octal-SPIRAM"
+        "SPIRAM_OCT": "Support for Octal-SPIRAM",
+        "FLASHSIZE_4MB": "4MiB flash size"
```

Comment:
Please use `"FLASH_4M": "4MiB flash"`, which matches the existing `ports/esp8266/boards/ESP8266_GENERIC/board.json`.

Reply:
thank you for your review!
changed flashsize_4mb to flash_4m